webpack/assets/javascripts/react_app/components/charts/StatisticsChartsList.js

```diff
@@ -1,111 +1,49 @@
-import React, {PropTypes} from 'react';
-import helpers from '../../common/helpers';
+import React, { PropTypes } from 'react';
 import chartService from '../../../services/statisticsChartService';
 import ChartBox from './ChartBox';
-import { STATUS } from '../../constants';
-import StatisticsStore from '../../stores/StatisticsStore';
-import StatisticsChartActions from '../../actions/StatisticsChartActions';
 import './StatisticsChartsListStyles.css';
+import { connect } from 'react-redux';
+import * as StatisticsChartActions from '../../redux/actions/statistics';
+import { STATUS } from '../../constants';
 
-class StatisticsChartsList extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {charts: this.stateSetup(this.props.data)};
-
-    helpers.bindMethods(this, [
-      'onChange',
-      'onError']
-    );
+const getStatusFromChart = (chart) => {
```

Comment:
this is a utility function. i dont think it belongs in the component, since it is not react-related.
anyway we should refactor this to avoid rendering the "smart" chartbox component.
after refactoring this function will go away.
i think it can also be in a utilities file, along with the translation strings.